src/components/Personality/PersonalityCard.jsx

```diff
@@ -99,6 +126,27 @@ class PersonalityCard extends Component {
                                         }
                                         href={`${this.props.hrefBase ||
                                             "personality/"}${personality._id}`}
+
+                                        style={{
+                                            background: "#2D77A3",
+                                            borderRadius: "30px",
+
+                                            position: "absolute",
+
+                                            height: "3em",
+                                            width: "8em",
```

Comment:
Width should be 100%, we don't need to have the exact same dimensions as the Figma mockup, just the proportions.